thrd_rcu/main.c

```diff
@@ -0,0 +1,75 @@
+#include <pthread.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "thrd_rcu.h"
+
+struct test {
+    unsigned int count;
+};
+
+static struct test __rcu *foo;
+
+static void *reader_side(void *argv)
```

Comment:
Rename to `reader` and `updater` for short.

Reply:
It will conflict with the name of `pthread_t` reader/updater.